analyses/cell-type-wilms-tumor-06/00_fetal_reference_kidney.Rmd

```diff
@@ -0,0 +1,240 @@
+---
+title: "Build azimuth compatible fetal kidney reference from the kidney cell atlas"
+author: "Maud PLASCHKA"
+date: '2024-08-07'
+output: 
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      message=FALSE,
+                      warnings=FALSE)
+```
+
+
+Introduction
+
+The aim is to build an azimuth compatible reference for fetal kidney from the kidney cell atlas.
+The rds data can be download using the download link https://datasets.cellxgene.cziscience.com/40ebb8e4-1a25-4a33-b8ff-02d1156e4e9b.rds
+
+## Packages
+
+Load required packages in the following chunk, if needed.
+Do not install packages here; only load them with the `library()` function.
+
+```{r packages, message=FALSE, warning=FALSE}
+library("Seurat")
+library(sctransform)
+library(Azimuth)
+library(SCpubr)
+library(tidyverse)
+library(patchwork)
+library(SeuratWrappers)
```

Comment:
I was poking around locally to see if I could better understand the Azimuth problem, and I think we need to get `SeuratWrappers` into the `renv.lock` file.

I was able to install from RStudio within the Docker container with:

```
remotes::install_github("satijalab/seurat-wrappers@8d46d6c47c089e193fe5c02a8c23970715918aa9")
```

(This is the most recent commit.)

If you run `renv::snapshot()`, I expect some packages might get removed because the code in this branch doesn't account for what is getting added in #704. That would be okay, though; we'd need to resolve it in whichever branch gets merged second.